### PR TITLE
Add badges

### PR DESCRIPTION
**Pull request by LuigiImVector**
## What does this PR do?
Improve repo

### Description

Added badges on README and CONTRIBUTING (see #5983)

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [X] Search for duplicates.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by davorpa** [comment]
I'd preffer all in the same site like in other repos

**Comment by LuigiImVector** [comment]
> I'd preffer all in the same site like in other repos.

@eshellman what do you think?

**Comment by eshellman** [comment]
I'm not a big fan of badges. The introduce a dependency.
For the CC license, if we want a badge we should use the one from Creative Commons. For the "Contributor Covenant", our CoC is "adapted from Contributor Covenant" but we don't use the words "Contributor Covenant" except on the CoC attribution, so I worry that the added language on the README might be confusing.
I would like to move the "Awesome" badge to below the fold. It's nice to have but oddly prominent on our page.

**Comment by LuigiImVector** [comment]
I removed the CoC badge to avoid confusion. The official CC license badge doesn't have the same height of "awesome" badge:

image

I think that the unofficial badge is ok but if it is not so, I kick it out and leave the "awesome" badge under the title.

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 *[Read this in other languages](../README.md#translations)*
 
-
```
Comment (davorpa) [inline docs/CONTRIBUTING.md:3]: Empty line removed :wink:

**Pull request merged**